CMakeLists.txt

```diff
@@ -94,6 +94,9 @@ check_include_file("X11/X.h" HAVE_X11)
 
 if(HAVE_PCAP)
     target_link_libraries(axpbox pcap)
+    message("-- pcap enabled")
+else()
+    message("-- libpcap not found. Suggestion: apt install libpcap-dev")
```

Comment:
I believe it would be better not to include distro-specific information into the message - notifying the user that Ethernet has been disabled would fit more.